### ISSUE TITLE
My Sites:Preload site content when user navigates to the my sites page

### ISSUE THREAD

**Issue by sendhil** [wordpress-mobile/WordPress-Android#4242]
When the user navigates to the my sites page and taps on any item for the first time the item should open to pre loaded content. To respect cellular data usage we should probably only do this on wifi.

Parallel WPiOS Issue - https://github.com/wordpress-mobile/WordPress-iOS/issues/5537

**Comment by designsimply** [comment]
Moved to Prioritized Android Groundskeeping because the issue is labeled as high priority and because respecting cellular data usage is a good thing. ❤️

**Comment by planarvoid** [comment]
If I understand this correctly, it is only relevant for the first load of the app because after that we already cache the Reader/Notifications data once the user opens the tab. The data is not changed after you select a different site (because it's related to the user, not the site). I don't think that preloading the data is the best behaviour. I think it's ok to let the user wait for the data to load the first time they open the app.
If we decide to do this, we need to come up with a clean solution (some kind of Sync service that would handle the data loading) and that's out of scope of a groundskeeping issue so we'd need to have a project and design/plan this properly.  
What do you think @designsimply ?

**Comment by designsimply** [comment]
I agree it's out of scope for Groundskeeping and think it's worth a larger discussion.

**Comment by designsimply** [comment]
I believe this particular issue is more about creating a better experience for users overall in terms of reducing the time spent waiting for the My Sites screen to load and less about reducing data usage as a main objective, so I think I probably misdirected the conversation before by mentioning data usage the way I did. I believe the goal for this issue was to consider the wait times a user is faced with when loading a main screen such as My Sites while still respecting data usage at the same time. Based on comments from the same issue on the iOS side, we should consider other screens as well.

> I should be able to open any of the five main sections, and not have to wait for it to load.
— Source: https://github.com/wordpress-mobile/WordPress-iOS/issues/5537#issuecomment-227291907

I've removed the high priority label from this issue due to inactivity, and would like to recommend re-evaluating to see if we can make any updates similar to the ones made for iOS in the past as part of https://github.com/wordpress-mobile/WordPress-iOS/issues/5537. cc @bummytime
